localstack/services/infra.py

```diff
@@ -377,7 +392,8 @@ def start_infra(asynchronous=False, apis=None):
             # this is a bit of an ugly hack, but we need to make sure that we
```

Comment:
nit: I guess we could remove the "hack" part of the comment (as we now have a cleaner solution that does not depend on `sleep_forever`) - we could change the comment to `We need to make sure that ...`